Gaijinizer patch/Troops.txt

```diff
@@ -7731,26 +7735,30 @@ Sphinx [All Book] Blow#MTLed
 
 \\N[1]くんをつかまえたーー！！
 > CONTEXT: Troops/176/pages/0/23/Dialogue/enemy11
-bwahahaa! ー!!I caught \\N[1]-kun! !#MTLed
+Bwahahahaaaa!!
+
+I caught you, \\N[1]-kun!!#Ron
 > END STRING
 
 > BEGIN STRING
 口でして欲しいんだったっけ…？♡
 
 ボク、キスが絶妙だから、フェラも得意なんだよ♡
 > CONTEXT: Troops/176/pages/0/27/Dialogue/enemy11
-Do you want me to speak ...? Moth
-I'm also good at blowjob because I'm exquisitely
-kissed.#MTLed
+Do you want me to do it with my mouth...? ♡
+
+Because my kisses are so exquisite, my blowjobs
+are also exceptional.#Ron
 > END STRING
```

Comment:
There's nothing technically wrong with the meaning of the translation here, but I don't think the kinda dumb Sphinx would use such a complicated word as "exquisite" here and in the next set of words. 

![image](https://user-images.githubusercontent.com/48231102/78420794-b6b31c80-760f-11ea-8879-1e4ddbece0b1.png)

We should probably go with a different synonym here... "Superb" or "Perfect" are indeed two options, but feel free to do whatever you think is appropriate.